src/brain_memory.js

```diff
@@ -169,7 +169,9 @@ function cleanRooms() {
         delete Memory.rooms[name];
       }
       if (Memory.myRooms.indexOf(name) < 0) {
-        console.log(Game.time, `Deleting ${name} from memory, no myRoom ${JSON.stringify(Memory.rooms[name])}`);
+        if (Object.keys(Memory.rooms[name]).length > 0) {
```

Comment:
So, this idea is to only store `myRooms` in Memory. All other rooms can be stored in the heap and recalculated. This makes sure that we don't reach the memory limit.

Currently some other rooms are also stored in memory, but mostly with empty values.

Do you know a case where we want to have external rooms stored in memory?